lib/core/test/unit/Cardano/Wallet/Api/TypesSpec.hs

```diff
@@ -956,6 +969,58 @@ instance Arbitrary ApiEpochInfo where
     arbitrary = ApiEpochInfo <$> arbitrary <*> genUniformTime
     shrink _ = []
 
+instance Arbitrary ApiScript where
+    arbitrary = ApiScript . ApiT <$> arbitrary
+
+instance Arbitrary Script where
+    arbitrary = do
```

Comment:
Same comment about generators of arbitrary tree structures applies here as it did in cardano-addresses.

In fact, could the types and generators from cardano-addresses be re-used?

Reply:
did in https://github.com/input-output-hk/cardano-wallet/pull/2253/commits/e6c0510a40d5f99f12d63caa03b7654078fe3ae9